master/test/testutils/fixtures.go

```diff
@@ -173,13 +173,25 @@ func DefaultMasterConfig() (*config.Config, error) {
 
 // DefaultElasticConfig returns the default elastic config.
 func DefaultElasticConfig() model.LoggingConfig {
-	port, err := strconv.Atoi(os.Getenv("DET_INTEGRATION_ES_PORT"))
-	if err != nil {
-		panic("elastic config had non-numeric port")
+	host := os.Getenv("DET_INTEGRATION_ES_HOST")
```

Comment:
also a cool change but either (1) needs its own callout in the PR description or (2) should be separated into its own PR

Reply:
noted elsewhere, will do